apps/web/src/utils/routeUtils.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  AUTH_INVITATION_TOKEN_TYPE,
+  AUTH_RESET_TOKEN_TYPE,
+  TEMPLATES_EDIT_TEMPLATEID_TYPE,
+} from '../constants/routeParams';
+
+type URLParams = AUTH_INVITATION_TOKEN_TYPE | AUTH_RESET_TOKEN_TYPE | TEMPLATES_EDIT_TEMPLATEID_TYPE;
+
+const escapeRegExp = (string: string) => string.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&');
```

Comment:
What is the goal of this regular expression?

Reply:
I followed @LetItRock . I think it escapes special characters present in URL